### DISCUSSION TITLE
[Question] Changing game directory

### DISCUSSION THREAD

**Discussion by PadowYT2** [truckersmp-cli/truckersmp-cli#309] [Q&A] [answered]
I have my game installed on `/media/padow/sea/SteamLibrary/steamapps/common/Euro Truck Simulator 2`, how would I go on and use that files instead of downloading the same game in `$HOME/truckers-cli/Euro Truck Simulator 2`?

**Comment by Lucki** [comment, marked as answer]
See https://github.com/truckersmp-cli/truckersmp-cli#options
Short option | Long option | Description
-- | -- | --
`-g DIR` | `--gamedir DIR` | Choose a different directory for the game files<br>Default: `$XDG_DATA_HOME/truckersmp-cli/(Game name)/data`

~~~ sh
truckersmp-cli start ets2mp --gamedir "/media/padow/sea/SteamLibrary/steamapps/common/Euro Truck Simulator 2"
~~~

Keep in mind they have to be the Windows files and that reusing the files may confuse steam or steamcmd or both when an update is happening. We decided against possible conflicts with steam and **this specific setup is unsupported**.
